big_scape/data/partial_task.py

```diff
@@ -67,8 +62,8 @@ def get_input_data_state(gbks: list[GBK]) -> bs_enums.INPUT_TASK:
     gbk_table = DB.metadata.tables["gbk"]
```

Comment:
I cant comment on all lines somehow, but 'distance_count' variable above should be named smth else if its just the number of rows in gbk table?